pkg/ui/frontend/src/pages/analyze-labels.tsx

```diff
@@ -133,7 +132,7 @@ export default function AnalyzeLabels() {
     useState<MetricType>("uniqueValues");
   const [openRows, setOpenRows] = useState<Set<string>>(new Set());
 
-  const form = useForm<z.infer<typeof formSchema>>({
+  const form = useForm({
```

Comment:
This line (and line 156) are the only changes to the code. Everything else is either:

* My dev environment reordering imports using Prettier 
* The new UI build